src/ui/common/src/utils/operators.ts

```diff
@@ -140,7 +140,13 @@ export function normalizeOperator(op): Operator {
   return op;
 }
 
-export type GetOperatorResultResponse = ExecState;
+export type GetOperatorResultResponse = {
+  name: string;
+  description: string;
+  // TODO: Remove status and just have exec_state.
```

Comment:
I think we should be safe to do this now. Hopefully our typechecking has our back here :)